Twittimation/Commands/Like.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Twittimation.Http;
+using Twittimation.IO;
+using Twittimation.Twitter;
+
+namespace Twittimation.Commands
+{
+    public sealed class Like : Command
+    {
+        public override List<string> RequiredArgs { get; } = new List<string>() { "TweetId" };
+        public override List<string> OptionalArgs { get; } = new List<string>();
+        public override Optional<string> OptionalRepeatedArg { get; } = new Optional<string>();
+        public override string HelpInfo { get; } = "Likes the specified tweet.";
+        public override string ExtendedHelp => HelpInfo;
+
+        private readonly IStored<Credentials> _credentials;
+
+        public Like(IStored<Credentials> credentials)
+        {
+            _credentials = credentials;
+        }
+
+        protected override void Go(string[] args)
+        {
+            var credentials = _credentials.Get();
```

Comment:
Lines 30-37, and 40-56 look like a duplicate of the code in Tweet. Should be refactored out into a `TwitterClient` class or similar